plenum/common/transactions.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum, unique
+
+
+@unique
+class PlenumTransactions(Enum):
+    #  These numeric constants CANNOT be changed once they have been used,
+    #  because that would break backwards compatibility with the ledger
+    NODE = "0"
```

Comment:
Added a comment.
Also I made a change to use values starting from 100 for Sovrin transactions, so we reduce a risk of overlap.